src/time-manager/Interface.ts

```diff
@@ -0,0 +1,123 @@
+import { DB } from '../datastore/db';
+import { ITask, ITaskP } from '../types/ITask.js';
+import { TaskService } from '../datastore/task.service.js';
+
+export class Interface {
```

Comment:
```suggestion
export class TaskManagerService {
```
Interface is bad name for class, it's better to avoid it.